src/NServiceBus.RabbitMQ/MessageConverter.cs

```diff
@@ -118,5 +126,12 @@ static string ValueToString(object value)
 
 
         static ILog Logger = LogManager.GetLogger(typeof(MessageConverter));
+
+        const string delayHeader = "NServiceBus.Transport.RabbitMQ.DelayInSeconds";
+        const string xDeathHeader = "x-death";
+        const string xFirstDeathExchangeHeader = "x-first-death-exchange";
+        const string xFirstDeathQueueHeader = "x-first-death-queue";
+        const string xFirstDeathReasonHeader = "x-first-death-reason";
+        const string confirmationIdHeader = "NServiceBus.Transport.RabbitMQ.ConfirmationId";
```

Comment:
In https://github.com/Particular/NServiceBus.RabbitMQ/pull/461 the variables start with an upper case character. Is there a reason that a different convention is being used here?

Reply:
As discussed, this is because they are private fields here, but public fields in the other case.